content/docs/explanation/netzgrafik-editor/_index.en.md

```diff
@@ -7,75 +7,35 @@ description: "Open-source software developed by SBB CFF FFS and its integration
 
-The transversal objective of this feature is to make 2 open-source projects from 2 big infrastructure managers work along and cooperate with one another with the same goals.
+The transversal objective of this feature is to make 2 open-source projects from 2 railway big infrastructure managers work along and cooperate with one another with the same goal, which is to ensure a digital continuity on different time scales for railway operational studies.
 
 #### 1 - Integration in OSRD
 
-Therefore, NGE is integrated in the operational studies section of OSRD, in an [`iframe`](https://developer.mozilla.org/fr/docs/Web/HTML/Element/iframe). An alternative way to deal with the integration would have been to refactor NGE as [`web-components`](https://developer.mozilla.org/fr/docs/Web/API/Web_components), thus easy to import in OSRD, but this solution has been abandoned, because of the quantity of refactoring that would have been needed. This `iframe` points to `osrd-nge`, a minimalist Angular app that embed the actual NGE app. It can also be seen as a wrapper of the actual NGE app. `osrd-nge` then customizes the actual NGE app with specific parameters and functionalities:
-- a `standalone` flag:
-  - to tell NGE to disable all back-end interactions (no database on NGE side)
-  - to tell NGE to disable some UI components (authentication, project management, versioning system view, etc.)
-- implementation of a communication interface between OSRD and NGE ([`message_event`](https://developer.mozilla.org/fr/docs/Web/API/Window/message_event), [`eventListener`](https://developer.mozilla.org/fr/docs/Web/API/EventTarget/addEventListener) and [`postMessage`](https://developer.mozilla.org/fr/docs/Web/API/Window/postMessage)).
+For external use, a [build of NGE is available on NPM](https://www.npmjs.com/package/netzgrafik-frontend) and is published at each release. In addition to this, OSRD has developed a `standalone` version of NGE, integrated into the source code, which allows NGE to function without a back-end. Finally, to meet needs specific to OSRD, OSRD uses a [fork](https://github.com/osrd-project/netzgrafik-editor-frontend) of NGE (whose [build is also available on NPM](https://www.npmjs.com/package/@osrd-project/netzgrafik-frontend)), remaining as close as possible to the official directory.
 
-NGE is then able to get the OSRD timetable as soon as a change is made on OSRD side, and OSRD is able to get the changes made on NGE side.
+Despite the non-compatibility of the front-end technologies (ReactJS for OSRD and Angular for NGE), this build allows OSRD to integrate NGE within an [`iframe`](https://developer.mozilla.org/fr/docs/Web/HTML/Element/iframe). This `iframe` deploys a minimalist Angular application which then imports the real NGE application thanks to the imported build.
```

Comment:
"Despite using different JavaScript frameworks" perhaps?

The iframe doesn't contain any Angular app per se, it just instantiates the custom element.